tests/src/drivers/test_driver_key_management.c

```diff
@@ -229,14 +232,30 @@ psa_status_t mbedtls_test_opaque_export_key(
     const uint8_t *key, size_t key_length,
     uint8_t *data, size_t data_size, size_t *data_length )
 {
+    psa_status_t status = PSA_ERROR_CORRUPTION_DETECTED;
```

Comment:
We should not change `mbedtls_test_opaque_export_key`. `mbedtls_test_opaque_import_key` is not implemented in 2.28 thus nothing to do in fact in the opaque case.

Reply:
Automatic conflict resolver error. My bad for not checking that.